File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/org/openapitools/client/model/PlayerSkinURL.scala

package org.openapitools.client.model


case class PlayerSkinURL (
    /* The URL to the texture, must be in Mojang's domains. */
    _url: String
)
object PlayerSkinURL {
    def toStringBody(var_url: Object) =
        s"""
        | {
        | "url":$var_url
        | }
        """.stripMargin
}

File: generated-sources/php-ze-ph/mojang-sessions/src/App/DTO/PlayerSkinURL.php
<?php

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

/**
 * Wraps the URL to the texture and configures the player&#39;s model
 */
class PlayerSkinURL
{
    /**
     * The URL to the texture, must be in Mojang&#39;s domains.
     * @DTA\Data(field="url")
     * @DTA\Validator(name="Type", options={"type":"string"})
     * @var string
     */
    public $url;
}
